edx_arch_experiments/kafka_consumer/docs/decisions/0003_kafka_managed_hosting.rst

```diff
@@ -0,0 +1,47 @@
+Kafka Managed Hosting
+=====================
+
+Status
+------
+
+Provisional
+
+Context
+-------
+
+* Setting up a `Kafka`_-based event bus can be complex since it comes in multiple parts that all need to be deployed
+  and maintained together.
+
+* The 2U/edX team responsible for this new infrastructure does not have the capacity to manage this in house.
+
+.. _Kafka: https://kafka.apache.org/
+
+Decision
+--------
+
+.. note::
+
+    This decision is specific to edx.org and is not a requirement or recommendation of how other deployments in the Open edX community manage their brokers. However, the background may still be useful to other commity members.
+
+For edx.org, the initial deployment of `Kafka`_-based event bus will use the software as a servics (SaaS) provider `Confluent`_.
+
+.. _Confluent: https://www.confluent.io/
+
+Additional Background
+---------------------
+
+`Amazon MSK`_ is an AWS managed service that supplies the Apache Kafka core platform only.
+
+The `Confluent Platform`_ adds additional capabilities, some of which are only commercially available.
+
+  * `Schema Registry <https://www.confluent.io/product/confluent-platform/data-compatibility/>`__
+  * Monitoring and alerting capabilities (Commercial)
+  * Self-balancing clusters (Commercial)
+  * Tiered storage (Commercial) (future feature of Apache Kafka)
+  * Infinite retention (Cloud only?)
```

Comment:
Can you clarify why this is a question?

Reply:
Because I'm not sure if it is part of the Confluent Platform, or just part of Confluent Cloud, both of which are Commercial. I am asking Confluent and maybe I can firm up an answer.

Reply:
I have updated.